docs/topics/secured-mission-intro.adoc

```diff
@@ -3,14 +3,18 @@
 
 The SSO booster project contains:
 
-* the sources for the Greeting service, which is the one which we are going to to secure
-* a template file (`service.sso.yaml`) to stand up the SSO server
-* the Keycloak adapter configuration to secure the service
+* The sources for the Greeting service, which we are going to to secure
+* A template file (`service.sso.yaml`) to deploy the SSO server
+* The Keycloak adapter configuration to secure the service
 
-= Standing up {RHSSO}
-The `service.sso.yaml` file contains all OpenShift configuration items to stand up a pre-configured {RHSSO} server. The SSO server configuration has been simplified for the sake of this exercise and does provide an out-of-the-box configuration, with pre-configured users and security settings.
+= Deploying {RHSSO}
+The `service.sso.yaml` file contains:
 
-IMPORTANT: It is not recommended to use this SSO configuration in production. Specifically, the simplifications made to the booster security configuration impact the ability to use it in a production environment.
+* All OpenShift configuration items needed to stand up a preconfigured {RHSSO} server
+* A default SSO server configuration which has been simplified for this exercise
```

Comment:
In line 14, "which has been" is still passive voice.

Reply:
I think we can afford to use it here because the information who simplified it is not actually important IMO.

Reply:
"which has been simplified for this exercise" may not even be necessary. If the requirement is *use the default config*, then the first part of the line already says that.

Reply:
I agree with @astrong1. Just remove that bit.